### DISCUSSION TITLE
Linux installation

### DISCUSSION THREAD

**Discussion by Peter186** [alire-project/alire#1316] [Q&A]
The Linux installation instructions on https://alire.ada.dev/docs/#first-steps  say to add alr to the environment variable PATH, by running

    export PATH=<PATH_TO_EXTRACTED>/bin/:$PATH

Should this not be

    export PATH=<PATH_TO_EXTRACTED>:$PATH ?

Thanks and regards.

**Comment by mosteo** [comment]
No, because the zip file has the `alr` binary inside a `bin` folder. So by default if you extract honoring paths it will be as the instructions show.

Of course in the end the path must point to where you place the `alr` binary, that you can put wherever you want. There are no dependencies on other files in the zip.

**Comment by Peter186** [comment]
Alejandro, 

Doh, I can see that, now! 

Many thanks for taking the time to explain it to me.  

Peter

-----------------------------------------
On Thursday, 9 February 2023 11:04:22 GMT Alejandro R Mosteo wrote:

No, because the zip file has the alr binary inside a bin folder. So by default if you extract honoring paths it will be as the instructions show.
Of course in the end the path must point to where you place the alr binary, that you can put wherever you want. There are no dependencies on other files in the zip.
?
